android/quest/src/main/java/org/smartregister/fhircore/quest/ui/pdf/PdfLauncherFragment.kt

```diff
@@ -30,16 +30,16 @@ import org.hl7.fhir.r4.model.Binary
 import org.hl7.fhir.r4.model.QuestionnaireResponse
 import org.jetbrains.annotations.VisibleForTesting
 import org.smartregister.fhircore.engine.R
-import org.smartregister.fhircore.engine.configuration.QuestionnaireConfig
+import org.smartregister.fhircore.engine.configuration.PdfConfig
 import org.smartregister.fhircore.engine.pdf.HtmlPopulator
 import org.smartregister.fhircore.engine.util.extension.decodeJson
 import org.smartregister.fhircore.engine.util.extension.extractLogicalIdUuid
 
 /**
  * A fragment for generating and displaying a PDF based on a questionnaire response.
  *
- * This fragment uses the provided [QuestionnaireConfig] to retrieve a questionnaire response,
- * populate an HTML template with the response data, and generate a PDF.
+ * This fragment uses the provided [PdfConfig] to retrieve a questionnaire response, populate an
+ * HTML template with the response data, and generate a PDF.
  */
 @AndroidEntryPoint
 class PdfLauncherFragment : DialogFragment() {
```

Comment:
Can we render this through the nav graph? Do you consider it necessary to keep it in the back stack history of the main application navigation graph?

Reply:
But the DialogFragment has no visible view, it's purpose is to launch the print manager for pdf preview

Reply:
FYI: dialog fragments can be included in the navgraph